<?xml version="1.0"?>
<launch>
    <remap from="scene_point_cloud" to="/gilbreth/kinect_points"/>
    <node pkg="gilbreth_perception" name="segmentation_node" type="segmentation_node">
      <rosparam command="load" file="$(find gilbreth_perception)/config/parameters.yaml"/>
    </node>
    <node pkg="gilbreth_perception" name="recognition_node" type="recognition_node">
       <rosparam command="load" file="$(find gilbreth_perception)/config/model_list.yaml"/>
       <rosparam command="load" file="$(find gilbreth_perception)/config/parameters.yaml"/>
       <param name="package_path" value="$(find gilbreth_perception)"/>
    </node>
</launch>

